tests/integration/tables/default_environment.cpp

```diff
@@ -0,0 +1,40 @@
+
+/**
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed as defined on the LICENSE file found in the
+ *  root directory of this source tree.
+ */
+
+// Sanity check integration test for default_environment
+// Spec file: specs/windows/default_environment.table
+
+#include <osquery/tests/integration/tables/helper.h>
+
+namespace osquery {
+namespace table_tests {
+
+class defaultEnvironment : public testing::Test {
+ protected:
+  void SetUp() override {
+    setUpEnvironment();
+  }
+};
+
+TEST_F(defaultEnvironment, test_sanity) {
+  auto const data = execute_query("select * from default_environment");
+
+  ASSERT_GE(data.size(), 0ul);
+
+  ValidatatioMap row_map = {
+      {"variable", NormalType},
```

Comment:
Could env variable be empty string? If not, can we check it here with `NonEmptyString` ?

Reply:
I don't believe it can. I'll change it to `NonEmptyString` 👍